protoc-gen-swagger/genswagger/atlas_patch.go

```diff
@@ -508,4 +516,160 @@ func gatherRefs(v interface{}) map[string]struct{} {
 
 func refToName(ref string) string {
 	return strings.TrimPrefix(ref, "#/definitions/")
-}
\ No newline at end of file
+}
+
+func applyCustomAnnotations(defs spec.Definitions) spec.Definitions {
+	for k, v := range defs {
+		v.Description = v.Title + v.Description
+		v.Title = ""
+
+		if title, ok := getAnnotationValue(v.Description, titleAnnotation); ok {
```

Comment:
why not getting all annotation values that are starting from @ and composing a map? If you're concerned about possible risk that 'description/title' line has separate line that starts with @, I would consider this possibility as neglect.